packages/seal/src/types.ts

```diff
@@ -1,9 +1,17 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import type { ClientWithExtensions, Experimental_CoreClient } from '@mysten/sui/experimental';
+import type {
+	ClientWithExtensions,
+	Experimental_CoreClient,
+	Experimental_SuiClientTypes,
+} from '@mysten/sui/experimental';
 
 export type KeyCacheKey = `${string}:${string}`;
 export type SealCompatibleClient = ClientWithExtensions<{
-	core: Experimental_CoreClient;
+	core: Experimental_CoreClient & {
+		verifyZkLoginSignature: NonNullable<
+			Experimental_SuiClientTypes.TransportMethods['verifyZkLoginSignature']
```

Comment:
what is the implication of this? today we can use the standard sui client

Reply:
yes you can use the standard sui client. this change is needed to make SealCompatibleClient compatible with ZkLoginCompatibleClient